src/components/MenuItem.js

```diff
@@ -72,7 +72,6 @@ const defaultProps = {
     isAnonymousAction: false,
     isSmallAvatarSubscriptMenu: false,
     title: '',
-    numberOfLinesTitle: 1,
```

Comment:
I don't think we should be removing this prop. We should still retain the check, and instead set this to 0 from `TaskPreview`.  I think this would break `MenuItem` at a few places.